build_debian.sh

```diff
@@ -95,6 +104,13 @@ sudo LANG=C chroot $FILESYSTEM_ROOT mount proc /proc -t proc
 sudo cp files/apt/sources.list $FILESYSTEM_ROOT/etc/apt/
 sudo cp files/apt/apt.conf.d/{81norecommends,apt-{clean,gzip-indexes,no-languages}} $FILESYSTEM_ROOT/etc/apt/apt.conf.d/
 sudo LANG=C chroot $FILESYSTEM_ROOT bash -c 'apt-mark auto `apt-mark showmanual`'
+if [[ $CONFIGURED_ARCH == armhf || $CONFIGURED_ARCH == arm64 ]]; then
+    # Remove amd64 x86 repo as it causes common packages version mismatch
+    sudo sed -i '/debian-archive.trafficmanager.net/d' $FILESYSTEM_ROOT/etc/apt/sources.list
+else
+    # Remove arm related repos
+    sudo sed -i '/arch=armhf,arm64/d' $FILESYSTEM_ROOT/etc/apt/sources.list
+fi
```

Comment:
it is a little bit hard to maintain in the future. 

I suggest to change line 104, sudo cp files/apt/sources.list.${CONFIGURED_ARCH} $FILESYSTEM_ROOT/etc/apt/sources.list

Reply:
Done